### PR TITLE
Adds toolbar container, its tests and examples

### PR DESCRIPTION
**Pull request by ahennr**
Title says it all. Please review.

### CODE REVIEW

File: src/Toolbar/Toolbar.example.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { Button } from 'antd';
```
Comment (ahennr) [inline src/Toolbar/Toolbar.example.jsx]: Could be changed to `SimpleButton` of `react-geo` in https://github.com/terrestris/react-geo/pull/1 is merged.

**Comment by marcjansen** [comment]
LGTM, please merge.

File: src/Toolbar/Toolbar.example.jsx (hunk 1 above)
Comment (KaiVolland) [reply src/Toolbar/Toolbar.example.jsx]: Just merge it. I'll change this in #1 afterwards...

**Comment by ahennr** [comment]
Thx for review.

**Pull request merged**